ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java

```diff
@@ -254,26 +254,7 @@ private void createBanner() {
         listener.onAdFailedToLoad(AdRequest.ERROR_CODE_INTERNAL_ERROR);
       }
     } else {
-      View adView = null;
-      VungleNativeAd vungleMRECBanner = Vungle.getNativeAd(placementId, mAdConfig, playAdCallback);
-      if (vungleMRECBanner != null) {
-        adView = vungleMRECBanner.renderNativeView();
-      }
-
-      if (adView != null) {
-        Log.d(TAG, "display MREC:" + vungleMRECBanner.hashCode() + this);
-        if (vungleBannerAd != null) {
-          vungleBannerAd.setVungleMRECBanner(vungleMRECBanner);
```

Comment:
I think VungleBannerAd should be cleaned from VungleNativeView as well (or VungleBanner can even replace VungleBannerAd)

Reply:
OK, will dig into this part. Thanks!

Reply:
@alexander-gubov Hi, I've checked the code and VungleBannerAd can be replaced.
Now VungleManager manages VungleBannerAdapter directly. VungleBannerAdapter will hold a VungleBanner to control VungleBanner attach, detach and destroy.